server/channel_ctools_direct_http.rb

```diff
@@ -77,25 +77,27 @@ def runGetCToolsSession
     use_url = format_url "/session.json"
     post_body = "_username=#{@user}&_password=#{@password}"
 
-    msg = Thread.current.to_s
-    elapsed = Stopwatch.new(msg)
-    elapsed.start;
+    elapsed = Stopwatch.new(Thread.current.to_s+": "+use_url)
+    elapsed.start
 
     # It does not seem possible to turn off ssl check on osx :-(, so may need to use http or search server url for testing.
     @session_id = RestClient.post use_url, post_body,
                                   {:verify_ssl => true}
 
     # make sure to print the elapsed time for the renewal.
-    elapsed.stop;
-    logger.debug("#{self.class.to_s}:#{__method__}: get session id post: stopwatch: "+elapsed.pretty_summary)
+    elapsed.stop
+    logger.debug("#{self.class.to_s}:#{__method__}:#{__LINE__}: get session id post: stopwatch: "+elapsed.pretty_summary)
   end
 
   def do_request(request)
     # This will fail as soon as other query parameters are added since teh session_id is blindly added.
     #  We can fix it if that happens.
     url = format_url(request)+"?_sessionId=#{@session_id}"
+    elapsed = Stopwatch.new(Thread.current.to_s+": "+url)
+    elapsed.start
     response = RestClient.get url, :verify_ssl => true
-   # logger.debug "#{self.class.to_s}:#{__method__}: response: "+response.inspect
+    elapsed.stop
+    logger.debug("#{self.class.to_s}:#{__method__}:#{__LINE__}: response: stopwatch: "+elapsed.pretty_summary)
```

Comment:
The URL is saved when the Stopwatch is created so the call being made is identified by that.  The class/method/line is also printed out.  Overall I think it is easy to identify the call from the log message.

 Sample output is below:
D, [2016-01-27T10:24:31.788735 #28032] DEBUG -- : ChannelCToolsDirectHTTP:do_request:102: response: stopwatch: 0.042,0,#<Thread:0x007fc5fba0c238>: https://ctqasearch.dsc.umich.edu/direct/session/becomeuser/ralt.json?_sessionId=06555922-c8d8-4557-9ee2-6f29cd9ac315